setup.py

```diff
@@ -38,7 +38,7 @@ def get_version() -> str:
 ]
 
 extras["quality"] = [
-    "black==21.4b0",
+    "black~=22.0",
```

Comment:
It is updated no?

Reply:
Yes it is, sorry I glanced over that! 

Peeking at it more, *eventually* this CI/CD may fail due to us not pinning the versions of black like we do with the setup.py https://github.com/huggingface/huggingface_hub/blob/main/.github/workflows/python-api-quality.yaml#L19